internal/rest/rest_operation.go

```diff
@@ -268,7 +268,7 @@ func AviRestOperate(c *clients.AviClient, rest_ops []*utils.RestOp) error {
 			op.Err = fmt.Errorf("Unknown RestOp %v", op.Method)
 		}
 		if op.Err != nil {
-			utils.AviLog.Warnf(`RestOp method %v path %v tenant %v Obj %s returned err %s with response %s`,
+			utils.AviLog.Infof(`RestOp method %v path %v tenant %v Obj %s returned err %s with response %s`,
```

Comment:
@monotosh-avi : This code path is for handling error condition. Will that be ok to change Log level from "Warn" to "Info"? It might get ignored during debugging issue.

Reply:
added by mistake, reverted.

Reply:
Keeping warnf is probably ideal?